perf/load-incluster/istio-values.yaml

```diff
@@ -0,0 +1,26 @@
+global:
+  controlPlaneSecurityEnabled: true
+
+  mtls:
+    enabled: true
+  disablePolicyChecks: true
+
+  sidecarInjectorWebhook:
+    enabled: true
+    rewriteAppHTTPProbe: true
+
+pilot:
+  traceSampling: 100.0
```

Comment:
This is probably going to cause a performance hit, since it makes 100% of traces captured (default is 1%). Not super familiar with the tracing though

Reply:
agreed - I am running at 100% now, and things seem pretty slow (3k RPS maximum atm on a single box0.